source/simulator/assembly.cc

```diff
@@ -686,6 +672,17 @@ namespace aspect
                                                              advection_field);
     double global_max_velocity = get_maximal_velocity(old_solution);
 
+    const UpdateFlags update_flags = update_values |
+                                     update_gradients |
+                                     (advection_field.is_temperature() ? update_hessians : UpdateFlags(0)) |
+                                     update_quadrature_points |
+                                     update_JxW_values;
+
+    /* Because we can only get here in the continuous case, which never requires
+     * face integrals, we supply no face_update_flags and an invalid
+     * face_quadrature to the scratch object to reduce the initialization cost.
+     */
+    const UpdateFlags face_update_flags = UpdateFlags(0);
```

Comment:
Use `update_default` as the initializer. It happens to be zero, but let's try to use the symbolic name.